src/elements/content-explorer/MoreOptions.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { Button, DropdownMenu, IconButton } from '@box/blueprint-web';
+import { Ellipsis } from '@box/blueprint-web-assets/icons/Fill';
```

Comment:
do you mind adding some snapshots on components before and after using blueprints

Reply:
vrt was added under contextExplorer folder